go/vt/vtgate/schema/tracker.go

```diff
@@ -225,6 +297,47 @@ func (t *Tracker) updateTables(keyspace string, res *sqltypes.Result) {
 	}
 }
 
+func (t *Tracker) updatedViewSchema(th *discovery.TabletHealth) bool {
+	viewsUpdated := th.Stats.ViewSchemaChanged
+	views, err := sqltypes.BuildBindVariable(viewsUpdated)
+	if err != nil {
+		log.Errorf("failed to read updated views from TabletHealth: %v", err)
+		return false
+	}
+	bv := map[string]*querypb.BindVariable{"viewNames": views}
+	res, err := th.Conn.Execute(t.ctx, th.Target, mysql.FetchUpdatedViews, bv, 0, 0, nil)
+	if err != nil {
```

Comment:
Shouldn't we be using an RPC for it, so that it is easier to make changes to the schema and be backward compatible?

Reply:
That is the next step. A new PR will be raised for that. This feature is now tested e2e by doing the existing way of how table schema is retrieved.